roles/installer/defaults/main.yml

```
@@ -484,3 +484,14 @@ nginx_worker_processes: 1
 nginx_worker_connections: "{{ uwsgi_listen_queue_size }}"
 nginx_worker_cpu_affinity: 'auto'
 nginx_listen_queue_size: "{{ uwsgi_listen_queue_size }}"
+
+# metrics-utility (github.com/ansible/metrics-utility)
+_metrics_utility_enabled: "{{ metrics_utility_enabled | default('false') }}"
```

boolean value doesn't need to be quoted
```suggestion
_metrics_utility_enabled: "{{ metrics_utility_enabled | default(false) }}"
```